snapshot/storage/bolt_test.go

```diff
@@ -14,6 +16,21 @@ func TestMetastore(t *testing.T) {
 	})
 }
 
+// TestKidnConversion ensures we can blindly cast from protobuf types.
+func TestKindConversion(t *testing.T) {
```

Comment:
This only tests that the types can be casted, which is only guaranteeing they are both integer types (the compiler already guarantees this). This test would be more useful if it was pairing the proto and snapshot types and ensuring they can both be cast to each other and be equivalent.